src/components/ebay-combobox/index.js

```diff
@@ -0,0 +1,244 @@
+const markoWidgets = require('marko-widgets');
+const Expander = require('makeup-expander');
+const scrollKeyPreventer = require('makeup-prevent-scroll-keys');
+const emitAndFire = require('../../common/emit-and-fire');
+const eventUtils = require('../../common/event-utils');
+const processHtmlAttributes = require('../../common/html-attributes');
+const observer = require('../../common/property-observer');
+const template = require('./template.marko');
+
+const comboboxOptionsClass = 'combobox__options';
+const comboboxExpanderClass = 'combobox__control';
+const comboboxHostSelector = `.${comboboxExpanderClass} > input`;
+const comboboxBtnClass = 'combobox__control';
+const comboboxOptionSelector = '.combobox__option[role=option]';
+
+function getInitialState(input) {
+    const options = (input.options || []).map(option => {
+        const selected = option.selected;
+
+        return {
+            class: option.class,
+            value: option.value,
+            label: option.label,
+            selected: Boolean(selected),
+            htmlAttributes: processHtmlAttributes(option),
+            renderBody: option.renderBody
+        };
+    });
+
+    const selectedOption = options.filter(option => option.selected)[0] || options[0];
+
+    if (options.length > 0 && selectedOption.value === options[0].value) {
+        options[0].selected = true;
+    }
+
+    return {
+        name: input.name,
+        class: input.class,
+        options,
+        selected: selectedOption,
+        grow: input.grow,
+        disabled: input.disabled,
+        borderless: Boolean(input.borderless),
+        htmlAttributes: processHtmlAttributes(input)
+    };
+}
+
+function getTemplateData(state) {
+    const comboboxClass = ['combobox', state.class];
+    const btnClass = [comboboxBtnClass];
+    const optionsClass = [comboboxOptionsClass];
+
+    if (state.borderless) {
+        btnClass.push('combobox__control--borderless');
+    }
+
+    return {
+        class: comboboxClass,
+        btnClass,
+        optionsClass,
+        name: state.name,
+        selectedOption: state.selected,
+        options: state.options,
+        grow: state.grow,
+        disabled: state.disabled,
```

Comment:
Disabled does not seem to do anything in this component.

Reply:
I think it does here: https://github.com/eBay/ebayui-core/pull/209/files/3fb66418b28ce66c87a6246909619e326ac3844a#diff-854be3f0318a650eed139adb0e266c38R12